### DISCUSSION TITLE
Apple silicon Mac iOS Apps sound cannot be route to Multi-Output Device

### DISCUSSION THREAD

**Discussion by soimy** [ExistentialAudio/BlackHole#620] [Q&A] [answered]
Apple silicon Mac (M1 M1pro M2)s can run iOS App natively via Mac App store or sideloaded IPAs. Unlike normal mac apps, those iOS apps cannot output sounds to Multi-Output Device ( Which is common for streaming apps like OBS to capture sound from blackhole while outputting sound via mac speakers).

Helps needed, thx.

**Comment by devinroth** [comment, marked as answer]
The answer is complicated. My suggestion is to set the output to BlackHole and then route the input of BlackHole to your speakers and in OBS set the input to BlackHole. You can route the input of BlackHole to your speakers with a DAW using input monitoring. Needlessly complicated but that's the solution.

**Comment by soimy** [comment]
Thanks for the instructions! It worked.
Just set the system audio output to Blackhole, in OBS set desktop audio source to blackhole, then route OBS audio monitoring to speakers/headphone.
